bufstream/docker-compose-quickstart/bufstream.yaml

```diff
@@ -0,0 +1,18 @@
+auto_migrate_metadata_storage: true
+cluster: bufstream
```

Comment:
Do we default `cluster` and `zone` yet? @jchadwick-buf I assume not, but just checking. If not, then keep for now obviously.